File: Codeforces/C++/problemset/Team.cpp
// Problem Link: https://codeforces.com/problemset/problem/231/A

#include <iostream>
using namespace std;
typedef long long ll;
int main()
{
    ll n,x,y,z,a = 0;
    cin >> n;
    for(int i = 0; i < n; i++)
    {
        cin >> x >> y >> z;
        if((x+y+z) >= 2)
            a++;
    }
    cout << a;
    return 0;
}

File: Codeforces/Kotlin/src/main/kotlin/problems/Team.kt
package problems.problems

/* Problem Link: https://codeforces.com/problemset/problem/231/A */

fun main() {
    val n = readln().toInt()
    var count = 0

    for(test in 1..n) {
        val votes = readln().split(" ").map { it.toInt() }

        if(votes[0] + votes[1] + votes[2] >= 2) {
            count++
        }
    }

    print(count)
}